GrandCentralBoard/Widgets/Bonus/Scene/BonusScene.swift

```diff
@@ -14,6 +14,7 @@ class BonusScene: SKScene {
     private var viewModel: BonusWidgetViewModel!
     private let world = SKNode()
     var bubbleResizeDuration: NSTimeInterval = 15
+    var relativeScenePadding: (x: CGFloat, y: CGFloat) = (x: 0.05, y: 0.05)
```

Comment:
I am not sure. It is not a size to use `CGSize`, not a vector for `CGVector`, and it is not a point. That's why I went with a tuple. I would prefer to stay with it, I think making it `CGPoint` would confuse someone reading the code.